Changed some doc links to use intersphinx.

jdufresne opened the pull request.

felixxm commented:
I'm not sure, tutorial fits better to How-to.

adamchainz commented on docs/howto/windows.txt:
```diff
@@ -57,8 +57,8 @@ It is best practice to provide a dedicated environment for each Django project
 documentation <https://packaging.python.org/guides/tool-recommendations/>`_.
-Python itself comes with `venv`_ for managing environments which we will use
-for this guide.
+Python itself comes with :mod:`venv` for managing environments which we will
+use for this guide.
```
```suggestion
Python itself comes with :doc:`python:tutorial/venv` for managing environments
which we will use for this guide.
```

jdufresne replied:
I've added this but also overrode the text. Sphinx's default uses the doc's title which is "Virtual Environments and Packages". The override is the original "venv".

jdufresne commented:
Thanks @felixxm. I totally overlooked that it wasn't linking to the module docs. I've incorporated the suggestion from @adamchainz. When doing so, I noticed another link that could use intersphinx so I updated that one too.

felixxm commented:
@jdufresne Thanks :+1: I would add `python:` e.g. ```doc:`venv <python:tutorial/venv>` ``` to make sure that links don't point to other docs.

adamchainz commented:
+1 to `python:` - explicit better than implicit 😉

jdufresne commented:
:+1: I added the `python:` prefix.
